### PR TITLE
conflict on @move must return http409 error

### PR DESCRIPTION
**Pull request by inakip**

### CODE REVIEW

File: guillotina/api/content.py [hunk 1]
```diff
@@ -583,6 +584,13 @@ def getRequestMethod(self):  # noqa
         raise ErrorResponse(
             "RequiredParam", _("Invalid params"), reason=error_reasons.REQUIRED_PARAM_MISSING, status=412
         )
+    except HTTPConflict:
+        raise ErrorResponse(
+            "Conflict Error",
+            "Id {data.get('new_id')} already exists in destination data.get('destination')",
+            reason=error_reasons.CONFLICT_ID,
+            status=409,
+        )
```
Comment (masipcat) [inline guillotina/api/content.py]: You don't need to capture the `HTTPConflict` and raise a `ErrorResponse`, the HTTPConflict isn't used to retry request (only `ConflictError` and `TIDConflictError` are retried): https://github.com/plone/guillotina/blob/fd73ae1bbe738d57eaa812b7eefe81d2914048ea/guillotina/asgi.py#L101-L108

Comment (inakip) [reply guillotina/api/content.py]: I see, thanks for the explanation :), fixing...

**Comment by bloodbare** [comment]
@vangheem I see ok now,

**Comment by vangheem** [comment]
@inakip can you also provide a PR for the 5.x branch?

**Comment by inakip** [comment]
> @inakip can you also provide a PR for the 5.x branch?

sure

**Comment by inakip** [comment]
@vangheem I need your approval since you requested changes. Also I need some admin to squash and merge since I do not have merge permission.

**Pull request merged**